modules/oadp-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,182 @@
+
+.CVE-2023-39325 Multiple HTTP/2 enabled web servers are vulnerable to a DDoS attack (Rapid Reset Attack)
+In previous releases of OADP, the HTTP/2 protocol was susceptible to a denial of service attack because request cancellation could reset multiple streams quickly. The server had to set up and tear down the streams while not hitting any server-side limit for the maximum number of active streams per connection. This resulted in a denial of service due to server resource consumption
+
+The following issues have been listed under this issue:
+
+* link:https://issues.redhat.com/browse/OADP-2886[OADP-2886]
+* link:https://issues.redhat.com/browse/OADP-2895[OADP-2895]
+* link:https://issues.redhat.com/browse/OADP-2869[OADP-2869]
+* link:https://issues.redhat.com/browse/OADP-2898[OADP-2898]
+* link:https://issues.redhat.com/browse/OADP-2904[OADP-2904]
+* link:https://issues.redhat.com/browse/OADP-2907[OADP-2907]
+* link:https://issues.redhat.com/browse/OADP-2910[OADP-2910]
+* link:https://issues.redhat.com/browse/OADP-2889[OADP-2889]
+* link:https://issues.redhat.com/browse/OADP-2892[OADP-2892]
+* link:https://issues.redhat.com/browse/OADP-2901[OADP-2901]
+
+For more information, see link:https://access.redhat.com/security/cve/cve-2023-39325[CVE-2023-39325 (Rapid Reset Attack)]
+
+
+For a complete list of all issues resolved in this release, see the list of link:https://issues.redhat.com/issues/?filter=12422837[OADP 1.3.0 resolved issues] in Jira.
+
+[id="known-issues-1-3-0_{context}"]
+== Known issues
+
+.CSI plugin panics on nil pointer when csiSnapshotTimeout is set to a short duration
+The CSI plugin panics on nil pointer when `csiSnapshotTimeout` is set to a short duration. Sometimes it succeeds to complete the snapshot within a short duration, but often it panics with the backup `PartiallyFailed` with the following error: `plugin panicked: runtime error: invalid memory address or nil pointer dereference`. link:https://issues.redhat.com/browse/OADP-3069[OADP-3069]
+
+.Backup is marked as PartiallyFailed when volumeSnapshotContent CR has an error
+If any of the `VolumeSnapshotContent` CRs have an error related to removing the `VolumeSnapshotBeingCreated` annotation, it moves the backup to the `WaitingForPluginOperationsPartiallyFailed` phase. link:https://issues.redhat.com/browse/OADP-2871[OADP-2871]
+
+.Performance issues when restoring 30k resources for the first time
+When restoring 30k resources for the first time, without an existing-resource-policy, it takes twice as long to restore them, than it takes during the second and third try with an existing-resource-policy set to `update`. link:https://issues.redhat.com/browse/OADP-3071[OADP-3071]
+
+.Post Restore Hooks might start running before Datadownload operation has released the related PV
```

Comment:
Post Restore Hooks - is this a terminology?

Reply:
It's not a proper name, so it still should be in sentence case.